Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -16,10 +16,67 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum <possumfromspace@gmail.com>
  */
 final class ExtraEmptyLinesFixer extends AbstractFixer
 {
     /**
+     * @var string[]|null
+     */
+    private $configuration = array(T_WHITESPACE => 'removeMultipleBlankLines');
+
+    /**
+     * @var Tokens
+     */
+    private $tokens;
+
+    /**
+     * Set configuration.
+     *
+     * Valid configuration option are:
+     * - 'break' remove blank lines after a line with a 'break' statement
+     * - 'continue' remove blank lines after a line with a 'break' statement
+     * - 'extra' [default] remove extra blank lines
+     * - 'return' remove blank lines after a line with a 'break' statement
+     * - 'throw' remove blank lines after a line with a 'break' statement
+     * - 'use' remove blank lines between 'use' import statements
+     *
+     * @param string[]|null $configuration
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            return;
+        }
+
+        $this->configuration = array();
+        foreach ($configuration as $item) {
+            switch ($item) {
+                case 'break':
+                    $this->configuration[T_BREAK] = 'removeEmptyLinesAfterLineWithTokenAt';
```

Comment:
I would keep that `token => method` configuration in some other structure and then keep `configuration` array as simple list of values (even simply the input array). I think it will be more readable - in configuration we will have user configuration of fixer, and in extra structure we will have fixer's internals

Sth like https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php#L22.